global.R

```diff
@@ -6,6 +6,8 @@ library(shinythemes)
 library(markdown)
 library(ggplot2)
 library(plotly)
+library(mvtnorm)

```

Comment:
```suggestion
library(mvtnorm)
library(rpact)
```
I added these two packages.